avm/res/db-for-my-sql/flexible-server/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,179 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-dbformysql.flexibleservers-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = 'eastus'
```

Comment:
Will be overwritten. Please refer to: https://github.com/Azure/bicep-registry-modules/pull/792#discussion_r1455072261

Reply:
set tempLocation